index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://github.com/bilalrajput09" class="name">
+        Bilal Ahmed
+      </a>
+        <img src="C:\Users\HP\My-Portfolio\img\threelines.png">
+    </div>  
+  </header>
+  <div class="introduction">
+    <h1>Hey There.
+      I’m Bilal Ahmed</h1>
```

Comment:
- [ ] Kindly 😊fix the font size of the h1 it should be like the Figma Design because yours is bigger than the Figma😊 

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212334493-0716c7d7-c9d3-4610-8bb5-b61a4dbaf1b7.png)  |   ![image](https://user-images.githubusercontent.com/83360636/212334443-0e4c80c1-18e4-4d56-84a7-c4354fa404a6.png)|